.github/workflows/build.yaml

```diff
@@ -19,3 +19,6 @@ jobs:
       - name: Build the CLI
         run: |
           make build-cli
+      - name: Build the container
+        run: |
+          make build-container
```

Comment:
Thanks - we'll want to run this container to verify it's actually working outside of the build process, but this is fine for now imo. We can also get that verification logic for free with the introduction of e2e tests, and I'm not super concerned with the container image breaking in the interim as it's not being used anywhere.

Reply:
This was fairly trivial to add so I just went ahead and did it! Also added the same "run" verification for the binary itself.